viewing_party/main.py

```diff
@@ -0,0 +1,292 @@
+
+#------------------------Wave 1--------------------------------------------------------------------------
+def create_movie(title, genre, rating):
+
+    #evaluate if any of these conditions are true then create a movie dictionary
+    if (title) and (genre) and (rating):
+        new_movie = {"title" : title, "genre" : genre,  "rating" : rating } 
+        return new_movie
+    
+    #if even one of the 3 criterias are not met then return none    
+    elif (not title) or (not genre) or (not rating):
+        return None
+    #for any other variety of critera too return none
+    else:
+        return None
+    
+def add_to_watched(user_data, new_movie):
+    #add the new movie to the users data's watched dictionary
+    user_data["watched"].append(new_movie)
+
+    return user_data
+
+def add_to_watchlist(user_data, movie):
+
+    #add movie to the user data's watchlist dctionary
+    user_data["watchlist"].append(movie)
+
+    return user_data
+
+def watch_movie(user_data, title):
+    #iterate over the user_data's watchlist (which is a dictionary)
+    for watchlist_movie in user_data["watchlist"]:
+        #evaluate if the title input is equal to the title in the watchlist dict for each movie
+        if title == watchlist_movie["title"]:
+            #if title is found then remove this movie from the users watchlist dictionary
+            user_data["watchlist"].remove(watchlist_movie)
+            #and add to the watched movie dictionary
+            user_data["watched"].append(watchlist_movie)
```

Comment:
Consider using `add_to_watched`
```suggestion
            add_to_watched(user_data, each_movie)
```